### PR TITLE
Improve assert message for lscpu get_cpu_info

### PR DESCRIPTION
**Pull request by dsrivastavv**

### CODE REVIEW

File: lisa/tools/lscpu.py [hunk 1]
```diff
@@ -230,11 +230,15 @@ def get_cpu_info(self) -> List[CPUInfo]:
         result = self.run("--extended=cpu,node,socket,cache").stdout
```
Comment (LiliDeng) [inline lisa/tools/lscpu.py]: should we assert the exit code?

**Comment by LiliDeng** [comment]
@somil55 please have a check on this PR when you have time, thanks!

File: lisa/tools/lscpu.py (hunk 1 above)
Comment (dsrivastavv) [reply lisa/tools/lscpu.py]: I gave it another thought, and I don't think we need assert exit code here. `lscpu` is a standard linux command and I haven't seen any failures on this command.

Comment (squirrelsc) [reply lisa/tools/lscpu.py]: > I gave it another thought, and I don't think we need assert exit code here. `lscpu` is a standard linux command and I haven't seen any failures on this command.

If it always succeeds, assert exit code should be fine. We cannot say something always right, so it doesn't need to be verified.

**Review by LiliDeng — APPROVED** [review]

**Pull request merged**